contracts/domainsSetLibrary.sol

```diff
@@ -0,0 +1,196 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+import "hardhat/console.sol";
+
+library domainSet {
+    struct Domain {
+        string name;
+        address owner;
+        uint256 price;
+        uint256 deposit;
```

Comment:
According to the task, there is no deposit anymore. But you have both the price and the deposit now.  -5 points